style.css

```diff
@@ -10,6 +10,7 @@ ul {
 
 a {
   text-decoration: none;
+  color: #fff;
 }
 
 @media (max-width: 450px) {
```

Comment:
- [ ] [optional] Please remove this media query because all the styles inside this only applied to screens below then 450px and it's breaking the design. We are supposed to apply these styles for all screens until 768px where our desktop styles start.


### See the image for reference

![image](https://user-images.githubusercontent.com/83803414/159937785-0f922376-6dc2-4a6e-9706-28d8dc467f65.png)